Description: src/luamm.hh: #include <functional>
 Required for building with gcc 7.
Author: Adrian Bunk <bunk@debian.org>
Bug-Debian: https://bugs.debian.org/853353

--- conky-1.10.6.orig/src/luamm.hh
+++ conky-1.10.6/src/luamm.hh
@@ -26,6 +26,7 @@
 #include <memory>
 #include <mutex>
 #include <exception>
+#include <functional>
 #include <stdexcept>
 #include <string>
 
